styles/BenefitsList.module.scss

```diff
@@ -0,0 +1,15 @@
+.container {
+    max-width: $content--narrow;
+    margin: 0 auto $component-spacing--large;
+    padding: 0 $gutter;
+}
+
+.benefits {
+    display: grid;
+    grid-template-columns: 1fr;
+
+    @include media-query(medium) {
+        gap: $gutter;
```

Comment:
comment: I’d have expected a gap for the mobile version as well? Judging by the code only, maybe there is spacing added elsewhere.